client/app/components/dialog-editor-modal/dialog-editor-modal.html

```diff
@@ -0,0 +1,108 @@
+<div class="modal-header">
+  <button type="button" class="close" ng-click="$dismiss()" aria-hidden="true">
+    <span class="pficon pficon-close"></span>
+  </button>
+  <h4 class="modal-title" id="myModalLabel" translate>Edit Dialog field</h4>
+</div>
+
+<div class="modal-body">
+  <ul class="nav nav-tabs" style="margin-bottom: 20px;">
```

Comment:
no go for inline styling

create sister sass file, `__dialog-editor-modal.sass`